Monika-After-Story/MonikaModDev
# New idle mode exps + fixes

### Changes:

- Monika has new special exp during idle mode.
- the `duration` property of `MASMoniIdleExp` can now be a `float`. When it's an `int`, we use `randint`, when it's a `float`, we use `uniform`. I needed that for better idle exp.
- changed the labels of `monika_brb_idle` and `monika_writing_idle` to be consistent with the rest. From now on all brb events should be of the `monika_idle_<type>` format, and brb callbacks of `monika_idle_<type>_callback`.
- slightly changed idle mode system. Before parts of it were scattered in our code. When we needed to update 1 thing, we had to update it in multiple places, which is an anti-pattern. Now to start idle mode, we call `mas_setupIdleMode` (the return key `"idle"` still works), to stop it, we call `mas_resetIdleMode`.
- brb callbacks can now return `"idle"` as well to keep idle mode.

### Testing:

- test the update script (at least via console, we'll test in typos anyway).
- try to use some brbs, verify everything works as expected.
- verify data for `monika_idle_brb` and `monika_idle_writing` was transferred successfully and they still work.
- check out the new idle mode exp. To test them you can: increase random chance (via console or via code) OR force it using its tag (via console). If you have suggestions, can req them or push.
